packages/gatsby-theme-styleguide/src/gatsby-components/Menu/MenuItem.js

```diff
@@ -57,24 +59,42 @@ const MenuLink = styled(Link)`
     `}
 `
 
-const MenuItem = ({ to, content, level, isPartOfCurrentlocation }) =>
-  to ? (
-    <MenuLink
-      to={to}
-      level={level}
-      isPartOfCurrentlocation={isPartOfCurrentlocation}
-    >
-      {content}
-    </MenuLink>
-  ) : (
-    <p>{content}</p>
-  )
+class MenuItem extends Component {
+  constructor(props) {
+    super(props)
+  }
+
+  render() {
+    let menuItem
+    if (this.props.to) {
+      menuItem = (
+        <MenuLink
+          to={this.props.to}
+          level={this.props.level}
+          isPartOfCurrentlocation={this.props.isPartOfCurrentlocation}
+          onClick={e => this.handleClick(this.props.hasChildren, e)}
+        >
+          {this.props.content}
+        </MenuLink>
+      )
+    } else {
+      menuItem = <p>{this.props.content}</p>
+    }
+
+    return menuItem
+  }
+
+  handleClick = (hasChildren, e) => {
```

Comment:
This method should be above the render method
```js
handleClick(e) {
  if (this.hasChildren) e.preventDefault()
}```

Reply:
fixed